stacks/spica/packages/core/route/route.ts

```diff
@@ -5,7 +5,8 @@ export enum RouteCategory {
   Primary = "Primary",
   Content = "Content",
   Developer = "Developer",
-  System = "System"
+  System = "System",
+  Functions = "Functions"
```

Comment:
```suggestion
  Function = "Function"
```
Shall we drop the plural suffix ?